### PR TITLE
https://issues.redhat.com/browse/ACM-11034 Update MSA KI

### PR DESCRIPTION
**Pull request by oafischer**
2.11 only

### CODE REVIEW

**Comment by zhiweiyin318** [comment]
Thanks @oafischer 
/lgtm

File: clusters/release_notes/known_issues.adoc [hunk 1]
```diff
@@ -266,26 +266,12 @@ You cannot use the {product-title-short} console to upgrade {ocp-short} managed
 The search details page for a certain resource on a certain managed cluster might fail. You must ensure that the work-manager add-on in the managed cluster is in `Available` status before you can search.
 
 [#non-ocp-logs]
-=== Non-{ocp} managed clusters require _ManagedServiceAccount_ or _LoadBalancer_ for pod logs after upgrade
-//2.4:15705
+=== Non-{ocp-short} managed clusters require _ManagedServiceAccount_ or _LoadBalancer_ for pod logs 
+//2.11:11034
 
-Both {ocp} and non-{ocp-short} clusters support the pod log feature if you are using a fresh install of {product-title-short} 2.10 or newer.
+The `ManagedServiceAccount` and cluster proxy add-ons are enabled by default in {product-title-short} version 2.10 and newer. If they are disabled after upgrading, you must enable the `ManagedServiceAccount` and cluster proxy add-ons manually to use the pod log feature on non-{ocp-short} managed clusters.
```
Comment (dockerymick) [inline clusters/release_notes/known_issues.adoc]:
```suggestion
The `ManagedServiceAccount` and cluster proxy add-ons are enabled by default in {product-title-short} version 2.10 and newer. If the add-ons are disabled after upgrading, you must enable the `ManagedServiceAccount` and cluster proxy add-ons manually to use the pod log feature on non-{ocp-short} managed clusters.
```

**Review by dockerymick — APPROVED** [review]
Made a suggestion, definitely preference. It is clear that when you say 'they', you are referring to the add-ons. This looks good to me

File: clusters/release_notes/known_issues.adoc [hunk 2]
```diff
@@ -266,26 +266,12 @@ You cannot use the {product-title-short} console to upgrade {ocp-short} managed
 The search details page for a certain resource on a certain managed cluster might fail. You must ensure that the work-manager add-on in the managed cluster is in `Available` status before you can search.
 
 [#non-ocp-logs]
-=== Non-{ocp} managed clusters require _ManagedServiceAccount_ or _LoadBalancer_ for pod logs after upgrade
-//2.4:15705
+=== Non-{ocp-short} managed clusters require _ManagedServiceAccount_ or _LoadBalancer_ for pod logs 
+//2.11:11034
 
-Both {ocp} and non-{ocp-short} clusters support the pod log feature if you are using a fresh install of {product-title-short} 2.10 or newer.
+The `ManagedServiceAccount` and cluster proxy add-ons are enabled by default in {product-title-short} version 2.10 and newer. If the add-ons are disabled after upgrading, you must enable the `ManagedServiceAccount` and cluster proxy add-ons manually to use the pod log feature on non-{ocp-short} managed clusters.
```
Comment (swopebe) [inline clusters/release_notes/known_issues.adoc:272]: @oafischer @zhiweiyin318 I am confused here. If this is not a 2.11 feature, but is a known issue, then the doc required should be removed from this 2.11 Story, which tells the doc team that we have to doc a new feature? Or 

Logically this is not tracking that workflow: https://issues.redhat.com/browse/ACM-10784

Is this technically a "removal" of something (going off the title here)

Comment (zhiweiyin318) [reply clusters/release_notes/known_issues.adoc:272]: The user only can use the log feature from the RHACM console->search page, there is no docs to describe this feature, only a known issue to describe the requirements. 
in 2.10 ACM supports 2 approaches to get logs on the backend, each approach has its requirements.  
in 2.11 we remove the old approach on the backend, so need remove the requirements of old approach from the know issue. 
The user cannot feel any change when using the log feature from the RHACM console.

**Pull request merged**